lib/omniauth/strategies/clever.rb

```diff
@@ -45,17 +45,73 @@ def callback_phase
       uid{ raw_info['data']['id'] }
 
       info do
-        { :user_type => raw_info['type'] }.merge! raw_info['data']
+        { :user_type => user_info['type'] }.merge! raw_info['data']
       end
 
       extra do
         {
           'raw_info' => raw_info
         }
       end
+      #  {
```

Comment:
I would put a newline above this comment block to more tightly associate it with this function and a comment explaining that this block is an example of the data shape returned.
I would similarly add a comment to the other call and it's data shape explaining that.